src/layer/annotation/index.ts

```diff
@@ -740,6 +748,24 @@ class RenderingOptionsTab extends Tab {
     super();
     const { element } = this;
     element.classList.add("neuroglancer-annotation-rendering-tab");
+
+    const colorControlToolWidget = element.appendChild(
```

Comment:
Can you please change this to insert the element right after the shader code instead please?